script/Ciel/Day 5/05_01_CIEL05_4B.txt

```diff
@@ -134,7 +134,7 @@ Ice spears pour down like raindrops.
 [sha:8da6f46c8786e937b3e283c9a77ee6dac26ab712]{
 -- Page 740, Offset 12862.
 -- 槍はすべて俺に向けられている。ヴローヴにとって、俺はもう<そ|・><う|・><い|・><う|・><モ|・><ノ|・>として格上げされた。
-The spears are all aimed at me now. Vlov has identified me as the greater threat.
+The spears are all aimed at me now. Vlov has identified me as the %{i}greater%{/i} threat.
```

Comment:
no need for italics

Reply:
or shift to "identified _me_ as the greater" if we feel like we have to